packages/nc-gui/pages/[projectType]/form/[viewId]/index/index.vue

```diff
@@ -165,10 +179,11 @@ const onDecode = async (scannedCodeValue: string) => {
                       <LazySmartsheetCell
                         v-else
                         v-model="formState[field.title]"
-                        class="nc-input"
+                        class="nc-input truncate"
                         :data-testid="`nc-form-input-cell-${field.label || field.title}`"
                         :class="`nc-form-input-${field.title?.replaceAll(' ', '')}`"
                         :column="field"
+                        :style="{ paddingLeft: '0.5rem !important' }"
```

Comment:
See the above comment. This line is not a correct fix.

Reply:
> See the above comment. This line is not a correct fix.

done , that CSS overriding wouldn't be happening if the style tag was scoped right ?

Reply:
FYI, `!` means `!important`. `!p-0` overrides every padding. Didn't see the reason we need that so removing that block should resolve the padding issue.

Reply:
ok got it , this is ready for merge right ?